### PR TITLE
WIP: Update for ePIX commissioning shift

### PR DESCRIPTION
**Pull request by ZLLentz**
WIP

### CODE REVIEW

File: ued/util.py [hunk 1]
```diff
@@ -39,5 +40,5 @@ def get_signal_by_pvname(pvname: str) -> ophyd.EpicsSignal:
             return sig
 
     if pvname not in _pv_cache:
-        _pv_cache[pvname] = Motor(pvname, name=pvname)
```
Comment (klauer) [inline ued/util.py]: 🤦‍♂️ @klauer

Comment (ZLLentz) [reply ued/util.py]: At least now you know that this was never used

Comment (ZLLentz) [reply ued/util.py]: They probably stuck to the same few known motors

**Comment by ZLLentz** [comment]
Dropping this PR but keeping the branch in case it's needed again for some reason

**Pull request closed without merging**